.github/workflows/build.yml

```diff
@@ -1,131 +1,99 @@
 # vim:ts=2:sw=2:et:ai:sts=2
 name: 'Build'
 
-on: ['push', 'pull_request']
+on:
+  push:
```

Comment:
@NightTsarina I missed the implications of this change. I think we might want to include `pull_request` here (I'll submit a PR).

Rationale: we have in the past set it to only push in repos in which we only create PRs out of branches in the same repo (not forks). This way CI for push was shared for PR, and there were no duplicate checks. But here, with forks as in #4, the checks don't run in this repo / don't show on the PR itself.

Unless you prefer some other configuration, I'll prepare a configuration with these semantics:

```
on:
  - pull_request
  - push
    - branches: main
```

Reply:
Ah, I haven't thought of that, thanks!